x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/optimizer/PhysicalPlanOptimizerTests.java

```diff
@@ -5698,9 +5749,9 @@ public void testPushTopNKeywordToSource() {
         var exchange = asRemoteExchange(topN.child());
```

Comment:
nit: for the tests in with changes in this file, it'd be nice to update the javadoc that contains the expected optimized plan. A simple way to obtain this is to debug the test in IntelliJ and look at `optimized` via the debugger.